.npmignore

```diff
@@ -0,0 +1,7 @@
+ui/
+tests/
+.gitignore
```

Comment:
Hmm ok. I didn't understand this from the [docs](https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package).